bin/travis/setupnode.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+npm install -g npm
```

Comment:
Just curious. You'are asking npm to install itself globally, right?

Reply:
There were some version differences, but after tests I can remove this line